quesma/table_resolver/table_resolver.go

```diff
@@ -278,10 +279,10 @@ func NewTableResolver(quesmaConf config.QuesmaConfiguration, discovery clickhous
 			decisionLadder: []basicResolver{
```

Comment:
Having this decisionLadder feels a bit overengineered to me. Why not just merge all these functions into one. This would improve readability IMO as we could read it from top to bottom

Reply:
We can fix it later. I just think we don’t need this level of generality.